bash_it.sh

```diff
@@ -51,32 +61,42 @@ done
 
 # Load theme, if a theme was set
 if [[ ! -z "${BASH_IT_THEME}" ]]; then
+  _log_debug "Loading theme..."
```

Comment:
Include the name of the theme in the message?

Reply:
done